packages/components/src/components/brandshape/brandshape.stories.ts

```diff
@@ -35,157 +42,90 @@ export default {
   }
 };
 
-const increaseColumnWidth = (): ConstantDefinition => {
-  return {
-    type: 'template',
-    name: 'width',
-    value: `<div style="min-width: 300px; max-width: 600px; width: 100vw;">%TEMPLATE%</div>`
-  };
-};
-
 /**
  * Default: This shows sd-brandshape in its default state.
  */
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- *  The different shapes of the brandshape.
+ * Use the `variant` attribute to display the brandshape with different background colors.
  */
-export const Shapes = {
-  name: 'Shapes',
-  parameters: { controls: { exclude: ['shapes'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'attribute',
-          name: 'shapes',
-          values: ['["top", "middle", "bottom"]', '["top", "middle"]', '["middle", "bottom"]', '["top"]']
-        }
-      },
-      args,
-      constants: increaseColumnWidth()
-    });
-  }
-};
 
-/**
- *  The different variants of the brandshape.
- */
-export const Variants = {
-  name: 'Variants',
-  parameters: { controls: { exclude: ['variant'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'attribute',
-          name: 'variant',
-          values: ['neutral-100', 'primary', 'white', 'border-primary', 'border-white', 'image']
-        }
-      },
-      options: {
-        templateBackgrounds: {
-          alternate: 'y',
-          colors: [
-            'white',
-            'white',
-            'rgb(var(--sd-color-primary, 0 53 142))',
-            'white',
-            'rgb(var(--sd-color-primary, 0 53 142))',
-            'white'
-          ]
-        }
-      },
-      args,
-      constants: increaseColumnWidth()
-    });
-  }
-};
+export const Variant = {
+  name: 'Variant',
+  render: () => html`
+    <div class="mb-5">
```

Comment:
Can you use one div for all elements with `space-y-5`?

Reply:
Updated